examples/js/loaders/SVGLoader.js

```diff
@@ -674,11 +698,11 @@ THREE.SVGLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 			var cxp = q * rx * y1p / ry;
 			var cyp = - q * ry * x1p / rx;
 
-			// Step 3: Compute (cx, cy) from (cx′, cy′)
+			// Step 3: Compute (cx, cy) from (cxâ², cyâ²)
 			var cx = Math.cos( x_axis_rotation ) * cxp - Math.sin( x_axis_rotation ) * cyp + ( start.x + end.x ) / 2;
 			var cy = Math.sin( x_axis_rotation ) * cxp + Math.cos( x_axis_rotation ) * cyp + ( start.y + end.y ) / 2;
 
-			// Step 4: Compute θ1 and Δθ
+			// Step 4: Compute Î¸1 and ÎÎ¸
```

Comment:
🤔

Reply:
What codification should have the source .js files? UTF-8 or other?

Reply:
Should be UTF-8. Maybe `modularize.js` doesn't currently support that?

Reply:
> Maybe `modularize.js` doesn't currently support that?

But the problem is in the .js, not the jsm.

Reply:
Just tested, and modularize seems to behave correctly.